metrics-core/src/main/java/software/amazon/swage/metrics/record/cloudwatch/CloudWatchRecorder.java

```diff
@@ -99,6 +99,14 @@
                 .build();
     }
 
+    private final Runnable sendAggregatedDataRunnable = () -> {
+        try {
+            this.sendAggregatedData();
+        } catch (Throwable t) {
+            log.fatal("CloudWatch recorder encountered an unrecoverable error and will stop", t);
```

Comment:
This should probably be at warning level, or at most error.  Problems in metric recording should not generally be assumed to be fatal to the application.

Reply:
This was the result of a discussion of the impact to applications when running without metrics, but I'll lower it to `error`.